metadata_backend/helpers/schemas/datacite.json

```diff
@@ -163,13 +163,9 @@
             "uniqueItems": true
         },
         "keywords": {
-            "type": "array",
+            "type": "string",
             "title": "Keywords",
-            "description": "A keyword or tag describing the resources. It is recommended to use a controlled vocabulary, ontology or classification when choosing keywords. At least one keyword is required.",
-            "items": {
-                "minLength": 1,
-                "type": "string"
-            }
+            "description": "A keyword or tag describing the resources. It is recommended to use a controlled vocabulary, ontology or classification when choosing keywords. At least one keyword is required."
```

Comment:
should this be specified as a comma-separated list

Reply:
yes i can add it.